priv/signs.json

```diff
@@ -1,4 +1,53 @@
 [
+  {
+    "id": "lab_test",
+    "type": "realtime",
+    "pa_ess_loc": "201",
+    "scu_id": "TBR01",
+    "read_loop_offset": 120,
+    "text_zone": "m",
+    "audio_zones": [
+      "m",
+      "n",
+      "s"
```

Comment:
I'm not sure if including _all_ of the supported zones is what we want here. Should it be only a subset?

Reply:
Just one is more typical, so let's use "m".